sdk/storagesync/Azure.ResourceManager.StorageSync/samples/Generated/Samples/Sample_CloudEndpointResource.cs

```diff
@@ -253,7 +253,7 @@ public async Task RestoreHeartbeat_CloudEndpointsRestoreheartbeat()
             CloudEndpointResource cloudEndpoint = client.GetCloudEndpointResource(cloudEndpointResourceId);
 
             // invoke the operation
-            await cloudEndpoint.RestoreHeartbeatAsync();
+            await cloudEndpoint.RestoreheartbeatAsync();
```

Comment:
mapping failed: https://github.com/Azure/azure-sdk-for-net/blob/91af14e56d8b9f12d66fc5217e2e13670a21d8e1/sdk/storagesync/Azure.ResourceManager.StorageSync/src/autorest.md?plain=1#L100